resources/application-connector/charts/application-registry/README.md

```diff
@@ -15,3 +15,5 @@ This service has the following parameters:
 - **specRequestTimeout** - A time-out for requests fetching specifications provided by the user. It is provided in seconds. The default time-out is `5`.
 - **assetstoreRequestTimeout** - A time-out for requests fetching specifications from the Asset Store Service. It is provided in seconds. The default time-out is `5`.
 - **detailedErrorResponse** - A flag for showing detailed internal error messages in response bodies. The default value is `false` and all internal server error messages are shortened to `Internal error`, while all other error messages are shown as usual.
+- **insecureAssetDownload** - A flag for skipping certificate verification for asset download.
+- **insecureSpecDownload** - A flag for skipping certificate verification for API specification download.
```

Comment:
The new parameter should also be documented in docs here https://github.com/kyma-project/kyma/blob/master/docs/application-connector/05-02-application-registry.md